FogCreek/Glitch-Community
# Onyx text: componentize team analytics

## Links
* Remix link: https://onyx-text.glitch.me
* Manuscript link: https://glitch.manuscript.com/f/cases/3328435/

## GIF/Screenshots:
![Screen Shot 2019-06-10 at 6 23 22 PM](https://user-images.githubusercontent.com/938722/59230905-04582780-8bad-11e9-9ebc-a1c8c299ba5a.png)
![Screen Shot 2019-06-10 at 6 23 45 PM](https://user-images.githubusercontent.com/938722/59230906-04582780-8bad-11e9-942b-0346fd0a77be.png)

## Changes:
* rebuild Team Analytics, using hooks, css modules, new popover components

## How To Test:
* visit https://onyx-text.glitch.me/@community while logged in
* the team analytics should render at the bottom of the page
* click 'All Projects' dropdown to filter analytics by project
* click 'Last 2 Weeks' dropdown to select time range
* click 'Remixes' to show remix analytics
* if a project is selected & remix tab is active, it should show project details

## Feedback I'm looking for:
in the project filter list, its a little confusing that we use the same color for currently selected, focused, and hovered rows: 
![Screen Shot 2019-06-10 at 6 45 23 PM](https://user-images.githubusercontent.com/938722/59231959-885fde80-8bb0-11e9-8a45-c87c493b03bb.png)
